app/assets/stylesheets/components/_discussion.css.sass

```diff
@@ -23,38 +26,129 @@ $discussion-item-triangle-size: 40px
+  text-align: center
+  
+.Discussion-author.Discussion-author--big
+  display: none
 
-.Discussion-commentsCount:before
-  content: ''
-  position: absolute
-  right: 0
-  bottom: 0
-  z-index: -1
-  width: 0
-  height: 0
-  border-style: solid
-  border-width: 0 0 $discussion-item-triangle-size $discussion-item-triangle-size
-  border-color: transparent
+.Discussion-commentsCount
+  @include delta
+  width: $discussion-inside-info-width
+  margin-top: 1px
+  color: theme-palette(neutral, white)
+  text-align: center
 
 .Discussion.Discussion--watching,
 .Discussion.Discussion--watched
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(primary)
+  .Discussion-commentsCount
+    background-color: theme-palette(primary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(primary)
 
 .Discussion.Discussion--unwatched
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(accent-secondary)
+  .Discussion-commentsCount
+    background-color: theme-palette(accent-secondary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(accent-secondary)
 
 .Discussion.Discussion--unread
-  .Discussion-commentsCount:before
-    border-bottom-color: theme-palette(accent-primary)
+  .Discussion-commentsCount
+    background-color: theme-palette(accent-primary)
+  .Discussion-afterCommentsCount
+    border-color: theme-palette(accent-primary)
+
+@media only screen and (min-width : 992px)
+  .Discussion
+    padding-right: 0.5rem
+
+  .Discussion:hover
+    .Discussion-commenterMarker
+      border-width: $discussion-marker-border-width*2
+
+    .Discussion-commenterLabel
+      transform: translateX(0)
+
+  .Discussion-commenters
+    position: absolute
+    top: 0
+    right: 100%
+    bottom: 0
+
+    display: block
+    overflow: hidden
+
+  .Discussion-commenter
+    display: flex
+    justify-content: flex-end
+
+    & + &
+      margin-top: 1px
+
+  .Discussion-commenterMarker
+    z-index: 2
+    border-right: $discussion-marker-border-width solid
+    transition: border-width 0.5s ease-in-out
+
+  .Discussion-commenterLabel
+    @include delta
+
+    flex: 1
+    padding: 0 $discussion-marker-border-width 0 2*$discussion-marker-border-width
+
+    background-color: theme-palette(neutral, white)
+
+    font-weight: bold
+    text-align: right
+    text-transform: uppercase
+    white-space: nowrap
+
+    transform: translateX(calc(#{$discussion-marker-border-width} + 100%))
+    transition: transform 0.5s ease-in-out
+
+  .Discussion-info
+    left: 100%
+
+    align-items: flex-start
+
+  .Discussion-author.Discussion-author--small
+    display: none
+
+  .Discussion-author.Discussion-author--big
+    display: block
+    border-left: 2*$discussion-marker-border-width solid
+    padding: 3px 6px
+
+    background-color: theme-palette(neutral, white)
+
+    font-weight: bold
+    text-transform: uppercase
+    white-space: nowrap
+
+  .Discussion-commentsCount
+    @include delta
```

Comment:
Isn't this already defined before in this component?
Things that are defined in the default state (without media queries), don't need to be set again within media queries.
Please take a look at the rest of the file and check if this is not happening with other properties